src/Constants.h

```diff
@@ -0,0 +1,31 @@
+#ifndef CONSTANTS
+#define CONSTANTS
+
+class Constants {
+  public:
+    enum class Mode {
+      POSITION_CONTROL, // Comands the servo to only respond to position-based input
+      VELOCITY_CONTROL, // Comands the servo to only respond to velocity-based input
+      COAST_CONTROL // Commands the servo to be unresponsive to any input
+    };
+
+    enum class Direction {
+        FORWARD, // Commands the servo motor to rotate in the counter-clockwise direction from a top-down perspective
+        BACKWARD // Commands the servo motor to rotate in the clockwise direction from a top-down perspective
```

Comment:
Clockwise and counter-clockwise are more intuitive than forward and backward for developers.

```suggestion
        COUNTER_CLOCK_WISE, // Commands the servo motor to rotate in the counter-clockwise direction from a top-down perspective
        CLOCK_WISE // Commands the servo motor to rotate in the clockwise direction from a top-down perspective
```

`CW` and `CCW` are also acceptable.